microsoftgraph/microsoft-graph-explorer-v4
# Fix: Add exception provision for spaces in query url

## Overview
Closes #1893 

### Demo
<img width="697" alt="image" src="https://user-images.githubusercontent.com/45680252/182403147-453259b2-ae53-4595-827f-13ad395cc40c.png">


### Notes

Optional. Ancillary topics, caveats, alternative strategies that didn't work out, anything else.

## Testing Instructions

* How to test this PR
On the query textbox, type a query like /me/drive/root:/New Folder
Notice that a whitespace error is not raised
Add a space between /me and /. Notice that a whitespace error is now displayed
